diff --git a/node_modules/react-native-gesture-handler/android/build.gradle b/node_modules/react-native-gesture-handler/android/build.gradle
index 20f41ea..343d0b7 100644
--- a/node_modules/react-native-gesture-handler/android/build.gradle
+++ b/node_modules/react-native-gesture-handler/android/build.gradle
@@ -132,6 +132,7 @@ repositories {
 
 android {
     compileSdkVersion safeExtGet("compileSdkVersion", 28)
+    namespace "com.swmansion.gesturehandler"
 
     // Used to override the NDK path/version on internal CI or by allowing
     // users to customize the NDK path/version from their root project (e.g. for M1 support)
@@ -142,6 +143,10 @@ android {
         ndkVersion rootProject.ext.ndkVersion
     }
 
+    buildFeatures {
+        buildConfig true
+    }
+
     if (REACT_NATIVE_MINOR_VERSION >= 71) {
         buildFeatures {
             prefab true
@@ -171,8 +176,8 @@ android {
     }
 
     compileOptions {
-        sourceCompatibility JavaVersion.VERSION_1_8
-        targetCompatibility JavaVersion.VERSION_1_8
+        sourceCompatibility JavaVersion.VERSION_11
+        targetCompatibility JavaVersion.VERSION_11
     }

+    kotlinOptions {
+        jvmTarget = JavaVersion.VERSION_11.toString()
+    }
 
     if (isNewArchitectureEnabled()) {
